test/testdata/golden/gapic/lang/csharp_gapic.yaml

```diff
@@ -0,0 +1,5 @@
+type: com.google.api.codegen.ConfigProto
```

Comment:
My general feeling is that if you have a problem, and you use `git submodule` to solve it, you usually have two problems.

This looks a lot like checking the Artman output against a baseline file. That is really an anti-pattern (toolkit notwithstanding). Can we not do that?